lib/Doctrine/DBAL/Migrations/Tools/Console/Command/VersionCommand.php

```diff
@@ -55,9 +55,11 @@ protected function configure()
             ->setName('migrations:version')
             ->setDescription('Manually add and delete migration versions from the version table.')
             ->addArgument('version', InputArgument::OPTIONAL, 'The version to add or delete.', null)
+            ->addArgument('version_end', InputArgument::OPTIONAL, 'The end version to add or delete. Works only with --range option.', null)
```

Comment:
Wouldn't that make more sense then to make a option of that argument ?
I have a hard time knowing when to use one or the other.